docs/docs/guides/how-to-create-syncs.mdx

```diff
@@ -0,0 +1,99 @@
+import ThreeDotMenu from './assets/sync-with-google-sheets-three-dot-menu.png';
+import CreateNewButtonModal from './assets/sync-with-google-sheets-create-modal.png';
+import CreateNewModal from './assets/sync-with-google-sheets-create-new-modal.png';
+import CreateNewModalFilled from './assets/sync-with-google-sheets-create-new-modal-filled.png';
+import EditDeleteSync from './assets/sync-with-google-sheets-edit-delete-options.png';
+import DashboardOverview from './assets/sync-with-google-sheets-dashboard-overview.png';
+import DashboardOverviewThreeDotMenu from './assets/sync-with-google-sheets-dashboard-overview-three-dot-menu.png';
+
+# Syncs
+
+Syncs let you send your chart results to a Google Sheet in your Google Drive on a schedule. You can set up as many syncs as you like, and if you make any changes to the content, Lightdash will update the syncs the next time they’re sent.
+
+## Enabling scheduled deliveries
+
+If you are running Lightdash on self-host, make sure you have set up Google Sheets integration before you start. See [setting up Google Sheets](self-host/customize-deployment/google-sheets-integration.mdx) for more information.
+If you're not on a self-hosted instance (e.g. Cloud or Enterprise), then your instance comes preconfigured with Google Sheets integration, so you don't need to do anything to enable it.
+
+## Creating a Sync
+
+Only users with [admin or edit access](/references/roles.mdx) can create Syncs. To create a Sync, click on the three-dot menu in the top-right corner of your chart, then click on `Sync with Google Sheets`.
+
+<img
+    src={ThreeDotMenu}
+    width="300"
+    height="276"
+    style={{ display: 'block', margin: '0 auto 20px auto' }}
+/>
+
+You'll get a modal that pops up and shows you the list of existing Syncs for your saved chart, and gives you the option to `Create new Sync`.
```

Comment:
The pop-up show you a list of any existing Syncs for the saved chart, you also have the option to `Create new Sync`.